wireguard-server/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:bullseye-slim
+
+RUN apt-get update && apt-get install -y wireguard iptables iproute2 curl systemd
```

Comment:
That's usually a good practice to also `apt upgrade` before installing anything
Also, is `systemd` required? I assume we might need it as a dependency for something else but not directly required.

Reply:
Added the apt upgrade before installing!

Reply:
About `systemd`: It'd definitely be cleaner to not install it -- I did so initially because the wgrest package uses it in a post-installation script (e.g. https://github.com/suquant/wgrest/blob/master/packaging/nfpm-amd64.yaml). Probably easy to avoid this. Any suggestions? :)

Reply:
No problem 👍 You can leave it as is
It was mostly me being curious about why we need systemd when we are in a container environment.
It will be hard to have `systemd` as PID 1 and I don't think it will run otherwise.
But it looks like we need it because of it creating a specific folder, so that's ok to install it 👍

Reply:
Ok will leave it as is for now then, but it sure is awkward to run systemd here :)

Thanks for all your help and feedback here, much appreciated!